scripts/Create-ChocoLicensePkg.ps1

```diff
@@ -50,8 +50,11 @@ Write-Warning "This script will OVERWRITE any existing license file you might ha
 & choco | Out-String -Stream | Write-Host
 Write-Warning "If there is is a note about invalid license above, you're going to run into issues."
 
+# Get license expiration date and node count
 [xml]$licenseXml = Get-Content -Path $LicensePath
 $licenseExpiration = [datetimeoffset]::Parse("$($licenseXml.SelectSingleNode('/license').expiration) +0")
+$licenseXml.license.name -match "(?<=\[).*(?=\])"
```

Comment:
```suggestion
$null = $licenseXml.license.name -match '(?<=\[).*(?=\])'
```

This will prevent `True` being emitted as output when the script runs. Not a show stopper, but doesn't really need output here.